terraform/lemp.tf.sample

```diff
@@ -7,65 +7,130 @@ module "lemp" {
   env      = "${var.env}"
   customer = "${var.customer}"
   deploy_bucket_name   = "${var.deploy_bucket_name}"
-
   #####################################
 
+  #. vpc_id (required):
+  #+ Amazon VPC id on which create each components.
   vpc_id                   = "<vpc-id>"
+
+  #. private_subnets_ids (required, array):
+  #+ Amazon subnets IDs on which create each components.
   private_subnets_ids      = ["private-subnets"]
-  public_subnets_ids       = ["<public-subnets>"]
+
+  #. public_subnets_ids (required, array):
+  #+ Amazon subnets IDs on which create each components.
+  public_subnets_ids      = ["public-subnets"]
+
+  #. bastion_sg_allow (optional):
+  #+ Amazon source security group ID which will be allowed to connect on Fronts port 22 (SSH).
   bastion_sg_allow         = "<bastion-sg>"
 
-  # metrics_sg_allow         = "<prometheus-sg>"
+  #. metrics_sg_allow (optional): ""
+  #+ Additionnal security group ID to assign to Cycloid workers. Goal is to allow monitoring server to query metrics.
```

Comment:
This doesn't seem to be for the `workers`, might want to update it, and regenerate the comments